### PR TITLE
Expand README

### PR DESCRIPTION
**Pull request by csoneson**
Adding some details to the README as I go through the process of dockerizing another app

### CODE REVIEW

**Review by hrhotz — APPROVED** [review]
perfect- thank you very much !

File: README.md [hunk 1]
```diff
@@ -1,49 +1,51 @@
...
+The shiny app should include an action button to "save and exit", which cleanly closes the app and stops the container.
 
 
-a simple shiny app in a docker container
+# Overview of the setup
 
+* The shiny app should be available in an R package. A template for such a package can be found [here](https://github.com/csoneson/templateDockerShinyPkg).
 
-# goal
+* The docker container is built on top of bioconductor/bioconductor_docker:RELEASE_3_19 and installs the package with the shiny app.
```
Comment (federicomarini) [inline README.md:11]: Is that 3_19 already provided? For now the `Dockerfile` still says 
image

Comment (csoneson) [reply README.md:11]: Yes, it's available - I forgot to change it here, sorry. I used the 3.19 one in https://github.com/csoneson/docker-exploremodelmatrix

Comment (federicomarini) [reply README.md:11]: Perfect, no big deal 😉 
Good to see it up and running already a few days after release!

**Pull request merged**